### PR TITLE
telemetry: Add EBS-account lookup via subscription_labels

### PR DESCRIPTION
**Pull request by wking**

### CODE REVIEW

**Comment by wking** [comment]
Adding a WIP header while I test this out.

**Comment by wking** [comment]
Rebased around #15

**Comment by wking** [comment]
Rebased onto #18, since it seems like we might be able to rustle up hosting with access to both the internal Telemetry service and the external Slack API.

**Comment by wking** [comment]
Pulled the WIP; this is currently live and working internally.

File: telemetry.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import requests
+
+
+__version__ = '0.0.1'
+URI = os.environ['TELEMETRY_URI']
+TOKEN = os.environ['TELEMETRY_TOKEN']
+
+
+_ADDITIONAL_REQUEST_ARGUMENTS = {}
+if 'TELEMETRY_CA_CERT' in os.environ:
```
Comment (gregsheremeta) [inline telemetry.py:13]: so if this isn't provided, does anything bad happen?

**Comment by gregsheremeta** [comment]
lgtm. Had a tiny question about the optional CA cert

File: telemetry.py (hunk 1 above)
Comment (wking) [reply telemetry.py:13]: > so if this isn't provided, does anything bad happen?

We just die on `CERTIFICATE_VERIFY_FAILED`.  But if you point your bot at a `TELEMETRY_URI` that does not require a custom CA, you don't need to set `TELEMETRY_CA_CERT`.

Comment (gregsheremeta) [reply telemetry.py:13]: makes sense. ty!

**Pull request merged**